src/schema/v2/me/myCollectionCreateArtworkMutation.ts

```diff
@@ -255,7 +255,15 @@ export const computeImageSources = (externalImageUrls) => {
   const imageSources = externalImageUrls.map((url) => {
     const match = url.match(externalUrlRegex)
 
-    if (!match) return
+    if (!match) {
+      if (url.startsWith("http")) {
```

Comment:
**thought:** do you intentionally have this as `http` instead of `https` ?

Reply:
Not really, my intention was to keep it as simple as possible and I don't see a need to restrict to `https`. We could check for `https` or even use a URL regex (for example `/^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&\/=]*)$/`)